MBBSEmu/Module/ModulePatch.cs

```diff
@@ -15,11 +16,18 @@ public enum EnumModulePatchType
 
         public string Name { get; set; }
         public string Description { get; set; }
-        public ushort Segment { get; set; }
-        public ushort Offset { get; set; }
+        public List<FarPtr> Addresses { get; set; }
+        public FarPtr Address { get; set; }
```

Comment:
seems a bit silly/error-prone to support a single Address as well as a List of them. How about just a List, and if you want one item, it's just a list of one item?